stdlib/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -59,14 +59,18 @@ public type Client client object {
     #
     # + sqlQuery - SQL statement to execute
     # + parameters - The parameters to be passed to the update query
+    # + returnGeneratedKeys - Indicate that generated keys should be made available for retrieval. If the value is
+    #                         set to true, auto-generated keys are made available to retrieve when the statement
+    #                         is executed. If the underline JDBC driver does not support it, still it will return `()`
```

Comment:
Shouldn't it be an error if the underline JDBC driver does not support it?

Reply:
+1. we need to send an error if underline JDBC not supported

Reply:
Yes, It returns an error  but I forgot to update here. 
I have fixed it.